include/powsybl/iidm/Switch.hpp

```diff
@@ -21,7 +20,7 @@ namespace iidm {
 
 class VoltageLevel;
 
-class Switch : public Identifiable, public MultiVariantObject {
+class Switch : public Identifiable {
```

Comment:
To be checked: does the implementation of MultiVariantObject methods ok (super call?)

Reply:
Ok, done by backporting Java PR #1476